docs/content/tutorial/assets/defining-an-asset.mdx

```diff
@@ -56,17 +56,17 @@ Serving dagit on http://127.0.0.1:3000 in process 70635
 
 You should be able to navigate to <http://127.0.0.1:3000> in your web browser and view your asset.
 
-<img src="/images/dagster/guides/asset-tutorial/defining-an-asset/defining_an_asset.png" />
+<img src="/images/guides/asset-tutorial/defining_an_asset.png" />
 
 Clicking on the "Materialize All" button will launch a run that will materialize the asset. After that run has completed, the shaded box underneath "cereals" holds information about that run. Clicking on the Run ID, which is the string of characters in the upper right of that box, will take you to a view that includes a structured stream of logs and events that occurred during its execution.
 
-<img src="/images/dagster/guides/asset-tutorial/defining-an-asset/asset_run.png" />
+<img src="/images/guides/asset-tutorial/asset_run.png" />
 
 In this view, you can filter and search through the logs corresponding to the run that's materializing your asset.
 
 To see a history of all the materializations for your asset, you can navigate to the _Asset Details_ page for it. Click the "cereals" link in the upper left corner of this run page, next to "Success". Another way to get to the same page is to navigate back to the Asset Graph page by clicking "Assets" in the top navigation pane, clicking on your asset, and then clicking on "View in Asset Catalog" at the top of the pane that shows up on the right.
```

Comment:
Revise for new UI. Eg (the activity tab is already shown above)

Click the "cereals" link in the upper left corner of this run page to go back to the asset details page.